<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HOD</class>
 <widget class="QDialog" name="HOD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Student</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Welcome..&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>500</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton{
font:bold;
background-color: rgb(217, 184, 255);
border-radius:5px;
border:1px solid;



}
#pushButton:pressed{
	background-color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>500</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_2{
font:bold;
background-color: rgb(217, 184, 255);
border-radius:5px;
border:1px solid;



}
#pushButton_2:pressed{
	background-color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Modify</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>190</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
background-color: rgb(188, 188, 188);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>190</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
background-color: rgb(188, 188, 188);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>190</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
background-color: rgb(188, 188, 188);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>190</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
background-color: rgb(188, 188, 188);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>190</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
background-color: rgb(188, 188, 188);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>190</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
background-color: rgb(188, 188, 188);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>600</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_3{
font:bold;
background-color: rgb(217, 184, 255);
border-radius:5px;
border:1px solid;



}
#pushButton_3:pressed{
	background-color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>560</y>
         <width>330</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
background-color: rgb(188, 188, 188);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>550</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_4{
font:bold;
background-color: rgb(217, 184, 255);
border-radius:5px;
border:1px solid;



}
#pushButton_4:pressed{
	background-color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>370</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
background-color: rgb(188, 188, 188);</string>
       </property>
       <item>
        <property name="text">
         <string>Computing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bussiness</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>420</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
background-color: rgb(188, 188, 188);</string>
       </property>
       <item>
        <property name="text">
         <string>Software Enginering</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Information Tecnolagy</string>
        </property>
       </item>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>60</y>
         <width>891</width>
         <height>451</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>1020</x>
         <y>520</y>
         <width>160</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_9{
font:bold;
background-color: rgb(217, 184, 255);
border-radius:5px;
border:1px solid;



}
#pushButton_9:pressed{
	background-color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Load Table</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Teacher</string>
      </attribute>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Welcome..&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>190</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
background-color: rgb(188, 188, 188);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>190</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
background-color: rgb(188, 188, 188);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>190</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
background-color: rgb(188, 188, 188);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>190</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
background-color: rgb(188, 188, 188);</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
background-color: rgb(188, 188, 188);</string>
       </property>
       <item>
        <property name="text">
         <string>Computing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bussines</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>400</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
background-color: rgb(188, 188, 188);</string>
       </property>
       <item>
        <property name="text">
         <string>Software Enginering</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Information Tecnolagy</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;Hello&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>480</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_5{
font:bold;
background-color: rgb(217, 184, 255);
border-radius:5px;
border:1px solid;



}
#pushButton_5:pressed{
	background-color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>480</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_6{
font:bold;
background-color: rgb(217, 184, 255);
border-radius:5px;
border:1px solid;



}
#pushButton_6:pressed{
	background-color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Modify</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>530</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_7{
font:bold;
background-color: rgb(217, 184, 255);
border-radius:5px;
border:1px solid;



}
#pushButton_7:pressed{
	background-color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_12">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>540</y>
         <width>260</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:5px;
background-color: rgb(188, 188, 188);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>590</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_8{
font:bold;
background-color: rgb(217, 184, 255);
border-radius:5px;
border:1px solid;



}
#pushButton_8:pressed{
	background-color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_2">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>30</y>
         <width>781</width>
         <height>451</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_10">
       <property name="geometry">
        <rect>
         <x>1050</x>
         <y>500</y>
         <width>130</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#pushButton_10{
font:bold;
background-color: rgb(217, 184, 255);
border-radius:5px;
border:1px solid;



}
#pushButton_10:pressed{
	background-color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Load Table</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
